assets/notifications.js

```diff
@@ -202,6 +203,8 @@ var Notifications = (function(options) {
         ret = $(html);
         ret.find('.notification-seen').click(function() {
             self.markSeen($(this).parents('.notification').data('id'));
+            $(this).parents('.notification').hide();
+            $('.notifications-icon-count, .notifications-header-count').text(parseInt($('.notifications-icon-count').html())-1);
```

Comment:
so that only updates whenever the poll is triggered, not if you mark a notification as seen, I have it setup where it only shows a seen checkmark and if that is clicked then it updates the count. I will work on making that not hard coded. I was in the mindset that this was specific to adminlte but can see the benefit in making it more transferable.